### PR TITLE
Remove unused peer dependency + format readme

### PR DESCRIPTION
**Pull request by Nexucis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,21 +4,41 @@ CodeMirror-promql
 [![NPM version](https://img.shields.io/npm/v/codemirror-promql.svg)](https://www.npmjs.org/package/codemirror-promql)
 
 ## Overview
-This project provides a mode for [CodeMirror Next](https://codemirror.net/6) that handles syntax highlighting, linting and autocompletion for the PromQL ([Prometheus Query Language](https://prometheus.io/docs/introduction/overview/)).
 
-> Initially this code was in a private repository. It has been transferred to the `prometheus-community` organization (Thanks to [Julius Volz](https://github.com/juliusv) who helped us with that).
-During the transfer, the repository and the package changed its name from `codemirror-mode-promql` to the current one: `codemirror-promql`.
+This project provides a mode for [CodeMirror Next](https://codemirror.net/6) that handles syntax highlighting, linting
+and autocompletion for the PromQL ([Prometheus Query Language](https://prometheus.io/docs/introduction/overview/)).
```
Comment (juliusv) [inline README.md:9]:
```suggestion
and autocompletion for PromQL ([Prometheus Query Language](https://prometheus.io/docs/introduction/overview/)).
```

File: README.md [hunk 2]
```diff
@@ -4,21 +4,41 @@ CodeMirror-promql
...
 ### Installation
-This mode is available as an npm package:
+
+This mode is available as a npm package:
+
+```bash
+npm install --save codemirror-promql
+```
+
+**Note:** You will have to manually install different packages part of [CodeMirror Next](https://codemirror.net/6), as
```
Comment (juliusv) [inline README.md:19]:
```suggestion
**Note:** You will have to manually install different packages that are part of [CodeMirror Next](https://codemirror.net/6), as
```

File: README.md [hunk 3]
```diff
@@ -4,21 +4,41 @@ CodeMirror-promql
...
+they are a peer dependency to this package. Here is the different packages you need to install:
```
Comment (juliusv) [inline README.md:20]:
```suggestion
they are a peer dependency to this package. Here are the different packages you need to install:
```

**Comment by juliusv** [comment]
:+1: besides comments

**Comment by Nexucis** [comment]
cool, thanks for your review !

**Pull request merged**